Packs/FireEyeETP/Integrations/FireEyeETP/FireEyeETP.py

```diff
@@ -43,6 +43,75 @@
 BASIC FUNCTIONS
 '''
 
+class Client(BaseClient):
+
+    def get_artifacts(self, alert_id):
+
+        url = f'/alerts/{alert_id}/downloadzip'
+        response = self._http_request(
+            method='POST',
+            url_suffix=url,
+            resp_type='response'
+        )
+        return response
+
+    def get_yara_rulesets(self, policy_uuid):
+
+        url = f'/policies/{policy_uuid}/configuration/rules/yara/rulesets'
+        response = self._http_request(
+            method='GET',
+            url_suffix=url
+        )
+        return response
+
+    def get_yara_file(self, policy_uuid, ruleset_uuid):
+
+        url = f'/policies/{policy_uuid}/configuration/rules/yara/rulesets/{ruleset_uuid}/file'
+        response = self._http_request(
+            method='GET',
+            url_suffix=url,
+            resp_type='response'
+        )
+        return response
+
+    def upload_yara_file(self, policy_uuid, ruleset_uuid, files):
+
+        url = f'/policies/{policy_uuid}/configuration/rules/yara/rulesets/{ruleset_uuid}/file'
+        response = self._http_request(
+            method='PUT',
+            headers={
```

Comment:
Can't you use the default client headers? with both content type and api key

Reply:
In some requests it is not possible to use the header that is established for the client, since the content type changes and it is not necessary to send it

Reply:
ok great thank you!